[
  {
    "poster": {
      "poster_name": "Aitor",
      "poster_handle": "aitor.tomas"
    },
    "time_stamp": "2024-10-25 01:02:28",
    "text": "Here it is the basic info of the debug:\n```Debugging info\n==============\nCLI version: 6.2.1\nnode version: v22.2.0\nplatform: macos\n\n[DEBUG] Reading config from /HubSpot-cms/HubSpot.config.yml\n[DEBUG] Attempting to track usage of \"project-upload\" command\n[DEBUG] Sending usage event to authenticated endpoint\n[DEBUG] Sent usage tracking command event: {\n  action: 'cli-command',\n  os: 'macos',\n  nodeVersion: 'v22.2.0',\n  nodeMajorVersion: 'v22',\n  version: '6.2.1',\n  command: 'project-upload',\n  authType: 'personalaccesskey',\n  type: 'STANDARD'\n}\n[DEBUG] Error: HubSpotHttpError: \n- message: undefined\n- status: undefined\n- statusText: undefined\n- method: undefined\n- code: undefined\n[DEBUG] Cause: HubSpotHttpError: The request in account XXXXXX was not found.\n    at [90m/HubSpot-cms/[39mnode_modules/[4m@HubSpot[24m/cli/node_modules/[4m@HubSpot[24m/local-dev-lib/http/index.js:23:27\n[90m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m\n    at async Axios.request [90m(/HubSpot-cms/[39mnode_modules/[4maxios[24m/dist/node/axios.cjs:4282:14[90m)[39m\n    at async ensureProjectExists [90m(/HubSpot-cms/[39mnode_modules/[4m@HubSpot[24m/cli/lib/projects.js:253:9[90m)[39m\n    at async exports.handler [90m(/HubSpot-cms/[39mnode_modules/[4m@HubSpot[24m/cli/commands/project/upload.js:47:3[90m)[39m {\n  cause: AxiosError: Request failed with status code 404\n      at settle [90m(/HubSpot-cms/[39mnode_modules/[4maxios[24m/dist/node/axios.cjs:2019:12[90m)[39m\n      at BrotliDecompress.handleStreamEnd [90m(/HubSpot-cms/[39mnode_modules/[4maxios[24m/dist/node/axios.cjs:3135:11[90m)[39m\n  [90m    at BrotliDecompress.emit (node:events:532:35)[39m\n  [90m    at endReadableNT (node:internal/streams/readable:1696:12)[39m\n  [90m    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)[39m {\n    code: [32m'ERR_BAD_REQUEST'[39m,```\n"
  },
  {
    "poster": {
      "poster_name": "Aitor",
      "poster_handle": "aitor.tomas"
    },
    "time_stamp": "2024-10-25 05:51:31",
    "text": "Ok, I managed to fix it by updating all NPM packages and starting from scratch.\nReally weird, it worked in one portal but it did not work in the other.\nAnyway, now it works in all portals."
  }
]